infrastructure/helm/quantumserverless/README.md

```diff
@@ -12,7 +12,13 @@ helm repo add kuberay https://ray-project.github.io/kuberay-helm
 ```shell
 helm dependency build
 ```
+Update values.yaml file. Find and replace the following strings
 
+- **CLIENTSECRET-CHANGEME**: string used as the secret for a OIDC protocol
+- **LOCAL-IP**: IP address that can be accessed from both outside of the cluster and inside of the cluster
```

Comment:
I would add information about how to obtain it locally, because it's true that can a bit tricky.